swagger.yaml

```diff
@@ -2609,6 +2667,72 @@ components:
           type: number
           format: double
 
+    getWorkdaysForUser:
+      type: object
+      required:
+      - date
+      - userGuid
+      - expectedHours
+      - enteredHours
+      - absenceHours
+      - incompleteTimingMinutes
+      - enteredTimeEntries
+      description: Workday details for a specific user
+      properties:
```

Comment:
These all look fine but we should also make sure we don't process more data than necessary. I would suggest checking with Marcel, if any of these properties are not actually going to be used in the front end then they should be removed from the spec and the lambda should map over the severa API response to ensure we do not send any unnecessary data.